Formula/woboq_codebrowser.rb

```diff
@@ -42,7 +42,6 @@ def install
 
     assert_predicate testpath/"test/test.c.html", :exist?
     assert_predicate testpath/"refs/printf", :exist?
-    assert_predicate testpath/"include/sys/stdio.h.html", :exist?
```

Comment:
`woboq_codebrowser` is having some trouble with non-standard SDK paths, i.e. starting Xcode 10, unless `macOS_SDK_headers_for_macOS_10.1{3,4}.pkg` is installed, system header files are no longer available under `/usr/include`.

I'm removing this line.